composer.json

```diff
@@ -46,7 +58,6 @@
   "extra": {
     "branch-alias": {
       "dev-main": "1.9.4.x-dev"
-    },
```

Comment:
unrelated change which needs to be avoided

Reply:
removed